src/System.Data.SqlClient/ref/System.Data.SqlClient.cs

```diff
@@ -116,6 +116,7 @@ public partial class SqlDataRecord
         public virtual void SetTimeSpan(int ordinal, System.TimeSpan value) { }
         public virtual void SetValue(int ordinal, object value) { }
         public virtual int SetValues(params object[] values) { throw null; }
+        System.Data.IDataReader System.Data.IDataRecord.GetData(int ordinal) { throw null; }
```

Comment:
The "public" modifier is implicit. I looked into other code parts in corefx and we don't have it everywhere